packages/account/src/providers/fuel-graphql-subscriber.ts

```diff
@@ -34,53 +34,66 @@ export class FuelGraphqlSubscriber implements AsyncIterator<unknown> {
     this.stream = response.body!.getReader();
   }
 
+  private events: Array<{ data: unknown; errors: unknown }> = [];
+  private parsingLeftover = '';
+
   async next(): Promise<IteratorResult<unknown, unknown>> {
     if (!this.stream) {
       await this.setStream();
     }
 
     // eslint-disable-next-line no-constant-condition
     while (true) {
+      if (this.events.length > 0) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const { data, errors } = this.events.shift()!;
```

Comment:
if `errors` is optional then I think we can remove the no-null assertion
```suggestion
        const { data, errors } = this.events.shift();
```

Reply:
This no-null assertion is related to [array.shift](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift) returning `T | undefined` because running `[].shift()` returns `undefined` so they had to cover that case in the type as well.